megamek/src/megamek/common/Crew.java

```diff
@@ -1097,10 +1097,17 @@ public void incrementFatigueCount() {
     }
     
     /**
-     * Sets fatigue counter back to zero.
+     * Sets crew state fields back to defaults. Used by MekHQ to clear game state.
      */
-    public void resetFatigue() {
+    public void resetGameState() {
         fatigueCount = 0;
+        doomed = false;
+        ejected = false;
+        for (int i = 0; i < crewType.getCrewSlots(); i++) {
+            unconscious[i] = false;
+            dead[i] = false;
```

Comment:
Do we really want to reset "dead" and "missing"? Or is the death of a particular crew member handled elsewhere?

Reply:
It's handled elsewhere. During scenario resolution this method gets called after crew casualties are applied to the Person, but before the crew is recalculated. That way we start with a blank slate, then any changes to the Crew are applied based on the availability and status of the MekHQ Person.

Reply:
Good to go, then.